OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -1000,6 +1000,23 @@ protected override void OnFirstRun(Actor self)
 					foreach (var cell in rallyPoint)
 						QueueChild(new AttackMoveActivity(self, () => mobile.MoveTo(cell, 1, evaluateNearestMovableCell: true, targetLineColor: Color.OrangeRed)));
 			}
+
+			public override IEnumerable<Target> GetTargets(Actor self)
+			{
+				if (ChildActivity != null)
+					return ChildActivity.GetTargets(self);
+
+				return Target.None;
+			}
+
+			public override IEnumerable<TargetLineNode> TargetLineNodes(Actor self)
+			{
+				var a = ChildActivity;
+				for (; a != null; a = a.NextActivity)
```

Comment:
This works, but it feels like a bandaid for what is probably a deeper issue (almost all other activities only consider the first child). I imagine this will want revisiting at some point in the future.

Reply:
this code is taken from here

https://github.com/OpenRA/OpenRA/blob/3111d06fcfc11130e435dabfe1d32ee860adbb75/OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs#L89-L98

it's not a bandaid. We simply don't need to render next activities for a child for any of the other activities